src/graphql/dataLoaders/articleCategoriesByCategoryIdLoaderFactory.js

```diff
@@ -0,0 +1,41 @@
+import DataLoader from 'dataloader';
+import client from 'util/client';
+
+export default () =>
+  new DataLoader(async categoryIds => {
+    const body = [];
+
+    categoryIds.forEach(id => {
+      body.push({ index: 'articles', type: 'doc' });
+
+      body.push({
```

Comment:
Not specifying size will cause it to return only 10 `articleCategories` by default. I think we need to handle pagination here?